Bottom: 44d69ae0b48efc4db810140148958af89f3cea29
Top:    84adb7c23c0d8aca265b38e4a57a64340f9953d9
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-04-12 23:01:00 -0400

Refresh of dma.patch

---

diff --git a/dt_config b/dt_config
index 5ac5df2..9a449e2 100644
--- a/dt_config
+++ b/dt_config
@@ -1126,7 +1126,7 @@ CONFIG_WATCHDOG=y
 # CONFIG_SOFT_WATCHDOG is not set
 # CONFIG_DW_WATCHDOG is not set
 # CONFIG_MAX63XX_WATCHDOG is not set
-CONFIG_LPC313X_WATCHDOG=y
+CONFIG_LPC31XX_WATCHDOG=y
 
 #
 # USB-based Watchdog Cards
